packages/cra-template-edu/template/.storybook/main.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Can we change this to be `.ts` with `import type { StorybookConfig } from '@storybook/core-common';` for a better developer experience

Reply:
Good idea. Seems to break the storybook build, though. I'll look into it some more.